### DISCUSSION TITLE
Emissives NBT Alternate Textures

### DISCUSSION THREAD

**Discussion by GreatKingKeenan** [MoreMcmeta/core#93] [Help Wanted]
Hi! I use the "nbt.display.Name" feature to add a few different variants to a Totem of Undying by changing the name of it in an anvil. I've gotten the different textures to load but the emissive overlays won't load as well. The base texture loads with it's emissive but not the alternate textures. I've tested it using Optifine and they work. I'd like to know if there's some special formatting I have to do to make the alternate textures also load their emissives with MoreMcmeta? Any help would be appreciated!

**Comment by soir20** [comment]
I think you're asking about the [custom item textures feature](https://optifine.readthedocs.io/cit.html), which is separate from emissives and isn't something MoreMcmeta supports. You would need to use a separate CIT mod. But MoreMcmeta emissives should work with the different CIT textures once they are actually being swapped.

**Reply by GreatKingKeenan** [reply]
Thank you for the quick response! Yeah, I'm using MoreMcmeta and Forge CIT. Forge CIT swaps the textures but doesn't work with MoreMcmeta to add the emissives. Here's the files if you need them for reference. The "textures/item" files work fine. The "optifine/cit" files that use the "custom item textures" feature are the ones that don't pick up the emissives.
[Torrezx-Halloween_totem(Modified).zip](https://github.com/user-attachments/files/16452778/Torrezx-Halloween_totem.Modified.zip)
